agent/acl.go

```diff
@@ -15,18 +15,15 @@ import (
 // critical purposes, such as logging. Therefore we interpret all errors as empty-string
 // so we can safely log it without handling non-critical errors at the usage site.
 func (a *Agent) aclAccessorID(secretID string) string {
-	ident, err := a.delegate.ResolveTokenToIdentity(secretID)
```

Comment:
Same concern here

Reply:
I see this method being called in two places: `Agent.filterMembers`, and `Agent.sendCoordinate`.

`Agent.sendCoordinate` is similar to the other case in `agent/local`. It's only done when we receive a permission denied error, and we will be included the accessorID in that error soon, so we should be able to remove that call entirely.

The second call `Agent.filterMembers` already has an `acl.Authorizer`. I missed this on the first pass! I'll push a commit which removes this call, and uses `authz.AccessorID()` to get the  value.